website_sale_wishlist_archive_cron/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+This module does the following:
+
+* A cron task that will "Archive" all the wishlist items that the product is inactive
```

Comment:
To reflect https://github.com/OCA/e-commerce/pull/801#issuecomment-1552670897...

```suggestion
This module introduces a scheduled action record that archives wishlist items associated with
inactive (archived) products.

Background:

The default behavior in vanilla Odoo for archived products is as follows:

* Existing wishlist items for the product remain unaffected (they persist in the wishlist).
* When a customer attempts to add the item to their cart, it vanishes from the wishlist and the cart
  remains unchanged.

This behavior can be confusing and may result in customer dissatisfaction.
```